### PR TITLE
Newest Libraries Page

### PR DESCRIPTION
**Pull request by jeff-zucker**
This should replace the two previous copies.  @Vinnl has kindly offered to change it's name if needed for the build process.  When you do, let me know the new name  and I'll change it in the public pointer for submitting PRs (the last line of the .md) which currently points to 2021-05-25_libraries.md.

### CODE REVIEW

File: _posts/developers/apps/2021-05-25_libraries.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+layout: for-developers
+title: Tools and libraries overview
+permalink: developers/tools
```
Comment (Vinnl) [inline _posts/developers/apps/2021-05-25_libraries.md:4]: So this line is what determines the final URL - and it's currently the same as 2019-01-01-00_overview.md, which is why I can't see it yet at https://deploy-preview-579--solidproject-dot-org.netlify.app/developers/tools

I think if you just delete the two files in  `_posts/developers/apps/tools/`, this should be good to go.

Comment (jeff-zucker) [reply _posts/developers/apps/2021-05-25_libraries.md]: Hmm, deleted the two old ones.  Now preview gives  a 404.  Maybe _overview.md is a magic name ... I'll try renaming.

Comment (jeff-zucker) [reply _posts/developers/apps/2021-05-25_libraries.md]: @Vinnl - sorry, unable to get anything other than a 404 in the preview.  I added the new file, deleted the two old files, then renamed the new file to overview.md ... still 404 in the preview

Comment (Vinnl) [reply _posts/developers/apps/2021-05-25_libraries.md]: Hmm... I'm kinda stumped too - as far as I can see this should do it. @justinwb do you see anything in the build log that might indicate why there's nothing at the above URL?

Comment (jeff-zucker) [reply _posts/developers/apps/2021-05-25_libraries.md]: closing this and starting with a clean PR

Comment (jeff-zucker) [reply _posts/developers/apps/2021-05-25_libraries.md]: Started fresh, same deal, moving the discussion to https://github.com/solid/solidproject.org/pull/580

**Pull request closed without merging**